repo2docker/app.py

```diff
@@ -741,7 +754,35 @@ def build(self):
                 shutil.rmtree(checkout_path, ignore_errors=True)
 
     def start(self):
-        self.build()
+        logfile = None
+        if (
+            self.s3_logs_endpoint
+            and self.s3_logs_access_key
+            and self.s3_logs_secret_key
+            and self.s3_logs_bucket
+        ):
+            logfile = tempfile.NamedTemporaryFile("w", delete=False)
```

Comment:
These builds logs have been useful in the past, maybe they should be kept anyway, even if not uploading to s3?